O/OpenMPI/build_tarballs.jl

```diff
@@ -20,44 +19,28 @@ script = raw"""
 # Enter the funzone
 cd ${WORKSPACE}/srcdir/openmpi-*
 
-if [[ "${target}" == aarch64-apple-* ]]; then
-    # Build internal `libevent` without `-Wl,--no-undefined`
-    # (taken from `L/libevent/build_tarballs.jl`)
+atomic_patch -p1 ../patches/0001-ompi-mca-sharedfp-sm-Include-missing-sys-stat.h-in-s.patch
 
-    # Expand libevent tarball so that we can patch it
-    pushd 3rd-party
-    tar xzf libevent-2.1.12-stable-ompi.tar.gz
-    cd libevent-2.1.12-stable-ompi
-    atomic_patch -p1 ${WORKSPACE}/srcdir/patches/build_with_no_undefined.patch
-    popd
+if [[ "${target}" == *-freebsd* ]]; then
+    # Help compiler find `complib/cl_types.h`
+    export CPPFLAGS="-I/opt/${target}/${target}/sys-root/include/infiniband"
 fi
 
-# Autotools doesn't add `${includedir}` as an include directory on some platforms
-export CPPFLAGS="-I${includedir}"
-
-# The configure scripts doesn't link against `libdl` by itself on many platforms
-export LIBS='-ldl'
-
 # We use `--enable-script-wrapper-compilers` to turn the compiler
 # wrappers (`mpicc` etc.) into scripts instead of binaries. As scripts,
 # they can be run in a cross-compiling environment, and cmake can
 # infer the MPI options. Otherwise, the MPI options need to be
 # specified manually for OpenMPI to work.
 
-./configure \
+./configure --prefix=${prefix} \
     --build=${MACHTYPE} \
-    --enable-mpi-fortran=usempif08 \
-    --enable-script-wrapper-compilers \
+    --host=${target} \
     --enable-shared=yes \
     --enable-static=no \
-    --host=${target} \
-    --prefix=${prefix} \
-    --docdir=/tmp \
-    --infodir=/tmp \
-    --mandir=/tmp \
-    --with-cross=${WORKSPACE}/srcdir/${target} \
-    --with-libevent=internal \
-    --without-cs-fs
+    --without-cs-fs \
+    --enable-mpi-fortran=usempif08 \
+    --enable-script-wrapper-compilers \
```

Comment:
Do we really want those xD

Reply:
We need `--without-cs-fs` for case-sensitive file systems.

We need both `--enable-mpi-fortran=usempif08` and `--enable-script-wrapper-compilers` for building other Yggdrasil packages.